nautobot/extras/views.py

```diff
@@ -2271,7 +2279,7 @@ def get_extra_context(self, request, instance):
         }
 
 
-class ObjectChangeLogView(generic.GenericView):
+class ObjectChangeLogView(generic.GenericView, UIComponentsMixin):
```

Comment:
1) Similar addition needed to `ObjectConfigContextView`? If not, why not?

2) Can we change the MRO to `(UIComponentsMixin, generic.GenericView)` here and below? If not, why not?